branchpro/__init__.py

```diff
@@ -18,4 +18,5 @@
 # Import main classes
 from .models import ForwardModel, BranchProModel    # noqa
 from .simulation import SimulationController  # noqa
+from ._sliders import _SliderComponent # noqa
```

Comment:
could you please move the sliders module into an apps module? I think it makes sense to have everything that is related to the apps in a separate folder. I.e. in seirmo an apps folder which has the _sliders.py and then we can add the simulation apps to that folder too.